src/GraphQL.Tests/Serialization/ExecutionErrorTests.cs

```diff
@@ -19,6 +19,29 @@ public void Simple(IGraphQLTextSerializer serializer)
         actual.ShouldBeCrossPlatJson(expected);
     }
 
+    [Theory]
+    [ClassData(typeof(GraphQLSerializersTestData))]
+    public void Extensions(IGraphQLTextSerializer serializer)
+    {
+        var error = new ExecutionError("some error 1")
+            .WithExtension("severity", "warn")
+            .WithExtension("rank", 42);
+
+        var expected = """
+{
```

Comment:
@Shane32 I would like to eventually switch all tests to use raw strings instead of `@` with `""`.